Source/WebKit/WebProcess/WebCoreSupport/WebPermissionController.h

```diff
@@ -62,6 +60,7 @@ class WebPermissionController final : public CanMakeWeakPtr<WebPermissionControl
         WebCore::ClientOrigin origin;
         WebCore::PermissionDescriptor descriptor;
         CompletionHandler<void(std::optional<WebCore::PermissionState>)> completionHandler;
+        WebPageProxyIdentifier identifier;
```

Comment:
I think this should come before the completion handler.

Reply:
Fixed!